cmd/burrow/commands/deploy.go

```diff
@@ -31,10 +31,10 @@ func Deploy(output Output) func(cmd *cli.Cmd) {
 		pathOpt := cmd.StringOpt("i dir", "", "root directory of app (will use pwd by default)")
 
 		defaultOutputOpt := cmd.StringOpt("o output", def.DefaultOutputFile,
-			"filename for jobs output file. by default, this name will reflect the name passed in on the optional [--file]")
+			"filename for playbook output file. by default, this name will reflect the playbook passed")
 
-		yamlPathOpt := cmd.StringOpt("f file", "deploy.yaml",
-			"path to package file which jobs should use. if also using the --dir flag, give the relative path to jobs file, which should be in the same directory")
+		playbooksOpt := cmd.StringsArg("FILE", []string{"deploy.yaml"},
```

Comment:
I think maybe it would be better to set the default as an empty array, then length check to give a helpful error message?

Reply:
Good point, thank you. Now the yaml file is a required argument.